### PR TITLE
Add -> product discount schema.

### PR DESCRIPTION
**Pull request by panshul007**
@salander  @hajoeichler please have a look

### CODE REVIEW

**Comment by hajoeichler** [comment]
@panshul007 please also add a minimal example.

File: schema/discount.schema.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          "value": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string"
+              },
+              "permyriad": {
+                "type": "integer"
+              },
+              "money": {
```
Comment (hajoeichler) [inline schema/discount.schema.json]: We should outsource the money type and reuse it here.

File: schema/discount.schema.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "$ref": "/ltext"
+          },
+          "description": {
+            "$ref": "/ltext"
+          },
+          "value": {
```
Comment (hajoeichler) [inline schema/discount.schema.json]: For the value sub document I would use a `oneOf` construct.

**Comment by panshul007** [comment]
@hajoeichler Please have a look if this is what u suggested. https://github.com/sphereio/sphere-json-schemas/commit/8280ab6867c086a494df71cbc453bac64795428a

**Pull request merged**